src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/SetAzureSqlServerAuditingPolicy.cs

```diff
@@ -42,12 +42,6 @@ public class SetAzureSqlServerAuditingPolicy : SqlDatabaseServerAuditingCmdletBa
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The set of the audit action groups")]
         public AuditActionGroups[] AuditActionGroup { get; set; }
 
-        /// <summary>
```

Comment:
So there is no working code that uses this parameter?  If so, this is fine.

Also, please document the changes in: https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Sql/ChangeLog.md, under 'current rrelease'